src/map/mob.c

```diff
@@ -1771,9 +1771,8 @@ static bool mob_ai_sub_hard(struct mob_data *md, int64 tick)
 		return true;
 
 	// Scan area for targets
-	if (!tbl && mode&MD_LOOTER && md->lootitem && DIFF_TICK(tick, md->ud.canact_tick) > 0
-	 && (md->lootitem_count < LOOTITEM_SIZE || battle_config.monster_loot_type != 1)
-	) {
+	if (battle_config.monster_loot_type != 1 && tbl == NULL && (mode & MD_LOOTER) != 0x0 && md->lootitem != NULL
```

Comment:
current option (if set to 0) allow loot items if mob already looted max allowed items. With value 1, looting can be if configured in mob db but only for max looting size.

Changed option, allow looting only if option is 0 and mob configured as looter. is this correct?
may be better change option to flag values? one bit for allow loot more than max value, other flag for disable looting at all, and may be other flags if you want something else?

Reply:
although I also support the name change for the battle_config,
but this should be an on/off behavior

currently 0 means yes (will loot) and 1 means no (will not loot)
which I support the name change, but set to the number of loots ?
I think currently max loot carry limit is 10 
-> #define LOOTITEM_SIZE 10 <- why this is set inside mapdefines.h ??
but poring will still continue to eats everything on the floor even if it already has loot more than 10

Reply:
see https://github.com/HerculesWS/Hercules/pull/2855#issuecomment-739331348